src/verification/verification.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@nestjs/common';
+
+@Injectable()
+export class VerificationService {
+    generateVerificationToken(userId: string): string {
+        // generate the token
```

Comment:
:wrench: Same as above

```suggestion
        // TODO: #36 - generate the token
```